bot/controller.py

```diff
@@ -61,4 +67,13 @@ def update_chatbot(self, bot_id, request: BotUpdate):
         except BotNotFound:
             raise HTTPException(status_code=400, detail="Bot not found")
         except Exception:
-            raise HTTPException(status_code=500, detail="Something went wrong")
+            raise HTTPException(status_code=500, detail=self.GENERAL_ERROR_MESSAGE)
+    
+    def delete_chatbot(self, bot_id):
+        try:
+            self.service.delete_chatbot(bot_id)
+            return {"detail": "Bot deleted successfully!"}
+        except BotNotFound:
+            raise HTTPException(status_code=400, detail="Bot not found")
+        except Exception:

```

Comment:
What kind of cases could potentially trigger the Exception error?

Reply:
One example would be when the id format is wrong, the error would trigger the general Exception error